controller_stopper/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+cmake_minimum_required(VERSION 3.5)
+project(controller_stopper)
```

Comment:
Depending on how well the solution sketched in #279 fulfills this it might make sense to include this, yes. However, in the long run I'd like to get rid of `ur_controllers` completely and make things as general as possible. For ROS1 we had something similar to the `controller_stopper` for KUKA RSI, as well, so my plan was, to have it accessible independent of any UR stuff. The state this is in currently, won't allow a completely independent adaption, however.